src/CenterSystem.java

```diff
@@ -185,12 +186,11 @@ public String editRecord(String managerId, String recordID, String fieldName, St
                     result = "fieldName doesn't match record type";
                     String operation = "edit: " + fieldName;
                     Log.log(Log.getCurrentTime(), managerId, operation, result);
-                } else{
-                    result = "No such record Id for this manager";
-                    Log.log(Log.getCurrentTime(), managerId, "edit: " + fieldName, result);
                 }
             }
         }
+        result = "No such record Id for this manager";
```

Comment:
since we have serverName, we can make:
"No such record for this manager on server n"
btw overall agreed on this placement of result update, previous result update placement was kinda error :)